tests/software_tests/can/transport_interface/test_python_can.py

```
@@ -26,85 +26,81 @@ class TestPyCanTransportInterface:
-        self._patcher_message = patch(f"{SCRIPT_LOCATION}.Message")
-        self.mock_message = self._patcher_message.start()
+        self._patcher_python_can_message = patch(f"{SCRIPT_LOCATION}.PythonCanMessage")
+        self.mock_python_can_message = self._patcher_python_can_message.start()
+        self._patcher_warn = patch(f"{SCRIPT_LOCATION}.warn")
+        self.mock_warn = self._patcher_warn.start()
+        self._patcher_time = patch(f"{SCRIPT_LOCATION}.time")
+        self.mock_time = self._patcher_time.start()
+        self._patcher_datetime = patch(f"{SCRIPT_LOCATION}.datetime")
+        self.mock_datetime = self._patcher_datetime.start()
+        self._patcher_wait_for = patch(f"{SCRIPT_LOCATION}.wait_for",
+                                       AsyncMock(side_effect=lambda *args, **kwargs: args[0]))
```

Ruff (0.12.2)

Unused lambda argument: `kwargs`

(ARG005)